source/MaterialXRender/ShaderValidators/Glsl/GlslValidator.cpp

```diff
@@ -533,7 +533,7 @@ void GlslValidator::save(const std::string& fileName)
     bindTarget(true);
     glPixelStorei(GL_PACK_ALIGNMENT, 1);
     glReadBuffer(GL_COLOR_ATTACHMENT0);
-    glReadPixels(0, 0, _frameBufferWidth, _frameBufferHeight, GL_RGBA, GL_FLOAT, buffer);
+    glReadPixels(0, 0, _frameBufferWidth, _frameBufferHeight, GL_RGBA, floatingPoint ? GL_FLOAT : GL_UNSIGNED_BYTE, buffer);
```

Comment:
Didn't have fixed point option before adding stb library. Now we can dump out fixed point so add option here.